packages/react/src/textarea/Textarea.css.ts

```diff
@@ -41,7 +41,9 @@ export const wrapper = recipe({
   ],
   variants: {
     resize: {
-      auto: {},
+      auto: style({
+        resize: "vertical",
```

Comment:
this should be "none"
```suggestion
        resize: "none",
```

Reply:
in fact you can move this to the base style (and make this and the `none` variant empty objects) and only in the vertical variant you can leave in the `resize: "vertical"` style